crates/wallet/src/wallet/mod.rs

```diff
@@ -1388,12 +1389,23 @@ impl Wallet {
         let mut received = Amount::ZERO;
 
         let recipients = params.recipients.iter().map(|(r, v)| (r, *v));
+        let mut contains_op_return = false;
 
         for (index, (script_pubkey, value)) in recipients.enumerate() {
             if !params.allow_dust && value.is_dust(script_pubkey) && !script_pubkey.is_op_return() {
                 return Err(CreateTxError::OutputBelowDustLimit(index));
             }
 
+            if script_pubkey.is_op_return() {
+                if contains_op_return {
+                    return Err(CreateTxError::MultipleOpReturn);
+                }
+                if script_pubkey.len() > MAX_OP_RETURN_BYTES {
+                    return Err(CreateTxError::MaxDataCarrierSize);
+                }
+                contains_op_return = true;
+            }
```

Comment:
Any reason to not use `is_standard_op_return` instead ?

Reply:
I don't think that is a member of `bitcoin 0.32.4`

Reply:
Oh, scratch that. Unfortunately, it's not available int that version.